scripts/copy-login-gov-keypair.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+###
+# Copies Login.gov JWT_KEY + JWT_CERT from one Cloud.gov application to another.
```

Comment:
This is simply a convenience script for developers, it is not called by any other methods. This is something I've had in my scratch files for awhile and have been using it to restore Login.gov credentials back to an application that has been deleted and re-created. Otherwise we would have to update the certificate in Login.gov and pass out the newly generated key to developers. Note that this does not replace the JWT rotation documentation, this is only suitable when it is desired to keep the current Login.gov keypair.